client/src/GettingStarted/GettingStarted.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react'
+import { Button, Card, Col, Row, Typography } from 'antd'
+import Icon from '@material-ui/core/Icon'
+import AssignmentIcon from '@material-ui/icons/Assignment'
+import BusinessIcon from '@material-ui/icons/Business'
+import CloudUploadIcon from '@material-ui/icons/CloudUpload'
+import PlaylistAddIcon from '@material-ui/icons/PlaylistAdd'
+
+export function GettingStarted() {
+  /*
+  TODO:
+  - layout/general
+    - ask about outer layout
```

Comment:
You can start the outer component as a part of this pull request if you want to, but you can also push it off and write another ticket, up to you depending on how much scope you want to cover here.

Reply:
Logged this as a separate task, #154. 👍